### PR TITLE
Bridgecrew CLI integration

### PR DESCRIPTION
**Pull request by tronxd**
Integrate checkov with bridgecrew's platform by a set of API calls, using a bridgecrew issued API key

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by schosterbarak** [comment]
can you please add markdown under docs/integrations too?

File: checkov/common/bridgecrew/platform_integration.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        persist_checks_results(reduced_scan_reports, self.s3_client, self.bucket, self.repo_path)
+
+    def commit_repository(self):
+        """
+        Finalize the repository's scanning in bridgecrew's platform.
+        """
+        request = None
+        try:
+            request = http.request("PUT", INTEGRATIONS_API_URL,
+                                   body=json.dumps({"path": self.repo_path, "branch": "master"}),
```
Comment (nimrodkor) [inline checkov/common/bridgecrew/platform_integration.py:110]: @tronxd Please add a todo to get the actual branch name!

File: checkov/main.py [hunk 2]
```diff
@@ -35,22 +36,35 @@ def run(banner = checkov_banner):
                         help='Report output format')
     parser.add_argument('-s', '--soft-fail',
                         help='Runs checks but suppresses error code', action='store_true')
+    parser.add_argument('--bc-api-key', help='Bridgecrew API key')
+    parser.add_argument('--repo-id',
+                        help='Identity string of the repository, with form <repo_owner>/<repo_name>')
```
Comment (nimrodkor) [inline checkov/main.py:41]: @tronxd Maybe add a branch parameter, which has the default value "master"?

Comment (tronxd) [reply checkov/main.py:41]: @schosterbarak WDYT?

Comment (schosterbarak) [reply checkov/main.py:41]: I agree. Branch should be a parameter. The SaaS should show results only for the default branch (usually master). 
maybe that's a value you can populate from circleci.

Comment (tronxd) [reply checkov/main.py:41]: @schosterbarak adding the branch as an arg to checkov, defaults to master.

**Pull request merged**